pest/src/error.rs

```diff
@@ -136,19 +136,25 @@ impl<R: RuleType> Error<R> {
     /// ```
     #[cfg_attr(feature = "cargo-clippy", allow(needless_pass_by_value))]
     pub fn new_from_span(variant: ErrorVariant<R>, span: Span) -> Error<R> {
-        let continued_line = if span.start_pos().line_col().0 != span.end_pos().line_col().0 {
-            Some(span.end_pos().line_of().to_owned())
+        let start = span.start_pos();
+        let mut end = span.end_pos();
+        if end.line_col().1 == 1 { // end position is after a \n
```

Comment:
This doesn't work on a `Span` from `0` to `0`.

Reply:
Oh, I didn’t think about this. Will fix!

Reply:
Hmm, I added a test, and it seems to work in some way